.github/workflows/run-e2e-tests-reusable-workflow.yml

```diff
@@ -844,12 +857,12 @@ jobs:
         env:
           SLACK_BOT_TOKEN: ${{ secrets.SLACK_BOT_TOKEN }}
         with:
-          channel-id: ${{ inputs.slack_notification_after_tests_channel_id }}
+          channel-id: ${{ inputs.slack_notification_after_tests_channel_id || secrets.SLACK_NOTIFICATION_AFTER_TESTS_CHANNEL_ID }}
           payload: |
             {
               "attachments": [
                 {
-                  "color": "${{ contains(join(needs.*.result, ','), 'failure') && '#C62828' || contains(join(needs.*.result, ','), 'cancelled') && '#FFA000' || '#2E7D32' }}",
+                  "color": "${{ contains(join(needs.*.result, ','), 'failure') && '#C62828' || contains(join(needs.*.result, ','), 'cancelled') && '#FFA000' || '#3fae55' }}",
```

Comment:
Why change the green hue? I don't care that much, just prefer we keep it consistent.

Reply:
@kalverra this is more in line with the rest of the notification content (emoji color). But, we can skip this

Reply:
@kalverra I reverted this as I see we use `#2E7D32` color in other places as well. Thanks for catching this!